### DISCUSSION TITLE
How to set NavBar Collapse

### DISCUSSION THREAD

**Discussion by ReneRam** [havit/Havit.Blazor#287] [Q&A]
Hi.
I went through the HAVIT Blazor documentation but can't find out how to have the **HxNavbar** collapse after a new link has been selected. I'm sure I'm missing something. Here's my code:

        `<HxNavbar ColorScheme="NavbarColorScheme.Dark" CssClass="my-background" >
        <HxNavbar CssClass="mb-3 my-background">
	        <HxNavbarBrand>
		        <img src="/images/small_logo.png" width="25" alt="Rome" />
		        Rome
	        </HxNavbarBrand>
        </HxNavbar>
            <HxNavbarToggler />
            <HxNavbarCollapse>
                <HxNav CssClass="me-auto mb-2 mb-lg-0">
                    <HxNavLink Href="/dashboard">Dashboard</HxNavLink>
                    <HxNavLink Href="/scheduler">Schedule</HxNavLink>
                    <HxNavLink Href="/eventlist">Events</HxNavLink>
                    <HxNavLink Href="/classes">Classes</HxNavLink>
                    <HxNavLink Href="/resourcelist">Resources</HxNavLink>
                    <HxNavLink Href="/profile">Profile</HxNavLink>
                    <HxNavLink Href="MicrosoftIdentity/Account/SignOut" >LogOut</HxNavLink>
                </HxNav>
            </HxNavbarCollapse>
        </HxNavbar>`

Anyone can help.
Thanks
ReneRam

**Comment by hakenr** [comment]
Hello @ReneRam,
do I understand correctly that you want the mobile (burger) version of the navbar to collapse when the item is clicked?

Currently, this has no built-in support from Bootstrap, nor Havit.Blazor.
I found this recipe https://stackoverflow.com/questions/42401606/how-to-hide-collapsible-bootstrap-navbar-on-click
(You can add the `data-bs-...` attributes to `HxNavLink` used and refer to the `HxNavbar` to collapse it when clicked.)

We will consider adding this feature into our components. @crdo?
